tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/journal/impl/AlignedJournalImplTest.java

```diff
@@ -532,6 +532,8 @@ public void testReloadInvalidCheckSizeOnTransaction() throws Exception {
 
       journalImpl.appendCommitRecord(1L, false);
 
+      journalImpl.debugWait();
+
       System.out.println("Files = " + factory.listFiles("tt"));
```

Comment:
It's meant to be... 

the appends are now asynchronous.. and we need to wait before we can make assertions.